generators/app/steps/CustomizableSteps/constants.js

```diff
@@ -18,40 +17,39 @@ module.exports.NPMRC_PATH = {
   destination: '.npmrc'
 };
 
-module.exports.REDUX_COMPONENTS = [`${COMPONENTS_PATH}/Field/index.js`];
-
 module.exports.FILES = [
   CONFIG_PATH,
   CONSTANTS_PATH,
   DOCS_README_PATH,
-  REDUX_PATH,
```

Comment:
Don't we need to change this for a global Context Folder.? Just for global values, like Auth

Reply:
global context is in ~app/context and it's already being added in this same object:

```
module.exports.FILES = [
  ...
  'src/app/reducer.ts',
  'src/app/context.ts',
  ...
]
```